### PR TITLE
chore(kommander): bump KCL to v0.3.4

### PR DESCRIPTION
**Pull request by jimmidyson**

### CODE REVIEW

File: stable/kommander/requirements.lock [hunk 1]
```diff
@@ -4,7 +4,7 @@ dependencies:
   version: 0.1.4
 - name: kommander-cluster-lifecycle
   repository: https://mesosphere.github.io/kommander-cluster-lifecycle/charts
-  version: v0.3.3
+  version: 0.3.4
```
Comment (hectorj2f) [inline stable/kommander/requirements.lock]: v0.3.4 ?

File: stable/kommander/values.yaml [hunk 2]
```diff
@@ -52,31 +52,9 @@ federate:
...
 
-  certificates:
-    issuer:
-      name:
-      kind: Issuer
-      selfSigned: false
-  konvoy:
-    allowUnofficialReleases: false
-  kubeaddonsRepository:
-    versionStrategy: repository-tag  # One of: `repository-tag`, `mapped-kubernetes-version`, `master`,
```
Comment (hectorj2f) [inline stable/kommander/values.yaml]: I'll miss these comments. Perhaps we have some documentation where they are explained :)

File: stable/kommander/requirements.lock (hunk 1 above)
Comment (jimmidyson) [reply stable/kommander/requirements.lock]: This is what `helm dependency update` did so I trust it, plus I've installed the chart and it works fine.

File: stable/kommander/values.yaml (hunk 2 above)
Comment (jimmidyson) [reply stable/kommander/values.yaml]: They are in the original chart.

File: stable/kommander/requirements.lock (hunk 1 above)
Comment (jimmidyson) [reply stable/kommander/requirements.lock]: All other dependencies are without the `v` prefix.

Comment (makkes) [reply stable/kommander/requirements.lock]: Yeah, I stumbled upon that every time. 🤷‍♂

**Pull request merged**